Kickstarter-iOS/Views/Controllers/BackerDashboardProjectsViewControllerTests.swift

```diff
@@ -0,0 +1,109 @@
+import Prelude
+import Result
+import XCTest
+@testable import Kickstarter_Framework
+@testable import KsApi
+@testable import Library
+
+// swiftlint:disable:next type_name
+internal final class BackerDashboardProjectsViewControllerTests: TestCase {
```

Comment:
oh wow, i was wondering why it felt so nice reviewing this pr and just realized that the test+implementation co-location means you alternate between the two when going down a pr! no more having tests all lumped together and implementation all together...